libbeat/tests/system/config/mockbeat.yml.j2

```diff
@@ -102,4 +102,12 @@ logging.metrics.period: {{ metrics_period }}
 keystore.path: {{keystore_path}}
 {% endif %}
 
+{% if xpack and xpack.monitoring -%}
+#================================ X-Pack Monitoring =====================================
+xpack.monitoring.elasticsearch.hosts: {{xpack.monitoring.elasticsearch.hosts}}
+xpack.monitoring.elasticsearch.metrics.period: 2s    # to speed up tests
```

Comment:
I would probably go even with 1s for both. Or do they have to be different?

Reply:
By default they are different, `10s` for metrics and `1m` for state so I wanted to mirror the difference, at least to some extent, in the test as well.

Perhaps I could make it `1s` for metrics and `2s` for state here?

Reply:
Let's keep it for now and in case we actually start to check for some differences in the tests, we can make adjustements.

Reply:
Okay, leaving values unchanged over here for now.